benchmark_db_terraform/modules/benchmark_db/service/run_benchmark.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'ruby_common/slack_client'
+require_relative 'ruby_common/utils'
+
+require 'logger'
+require 'fileutils'
+
+# Retrieves an environmental variable, failing if its not set or empty.
+def get_and_assert_env_variable(name)
+  var = ENV[name]
+  raise "Please set #{name} environmental variable" if var.nil? || var.empty?
+
+  var
+end
+
+SLACK_TOKEN = get_and_assert_env_variable 'SLACK_API_TOKEN'
+CHANNEL = get_and_assert_env_variable 'SLACK_NOTIF_CHANNEL'
+SCRIPTS_DIR = get_and_assert_env_variable 'BASE_FOLDER'
+LOG_DIR = get_and_assert_env_variable 'BASE_FOLDER'
+
+# Current datetime, to append to the log files
+DATE = Time.new.strftime '%FT%H:%M:%S'
+LOG_HEALTH = "#{LOG_DIR}/benchmark_#{DATE}_health"
+LOG_SYNC = "#{LOG_DIR}/benchmark_#{DATE}_sync"
+
+# Create log directory
+FileUtils.mkdir_p LOG_DIR
+
+logger = Logger.new(LOG_SYNC)
+
+health_check_passed = false
+
+# Install dependencies
+init_commands = <<CMD
+  dnf install -y aria2 zstd clang clang-devel cmake git bzr jq pkgconfig mesa-libOpenCL mesa-libOpenCL-devel opencl-headers ocl-icd ocl-icd-devel llvm wget hwloc hwloc-devel golang rust cargo s3cmd
+  dnf clean all
+  gem install sys-filesystem bundler concurrent-ruby deep_merge tomlrb toml-rb csv fileutils logger open3 optparse set tmpdir
+CMD
```

Comment:
Shouldn't these be part of the docker image?

Reply:
yes moved